cli/src/commands/auth.rs

```diff
@@ -65,18 +66,24 @@ pub async fn handle_auth_status(
 }
 
 /// Display the current authentication token to the user, if one exists.
-pub fn handle_auth_token(config: &Config) -> CommandResult {
-    match config.auth_info.offline_access.clone() {
-        Some(token) => {
-            println!("{}", token);
-            Ok(ExitCode::Ok.into())
-        }
+pub async fn handle_auth_token(config: &Config, matches: &clap::ArgMatches) -> CommandResult {
+    let refresh_token = match &config.auth_info.offline_access {
+        Some(refresh_token) => refresh_token,
         None => {
             print_user_warning!(
                 "User is not currently authenticated, please login with `phylum auth login`"
             );
-            Ok(ExitCode::NotAuthenticated.into())
+            return Ok(ExitCode::NotAuthenticated.into());
         }
+    };
+
+    if matches.is_present("bearer") {
+        let tokens = auth::handle_refresh_tokens(&config.auth_info, refresh_token).await?;
+        println!("{}", String::from(&tokens.access_token));
```

Comment:
It seems like `AccessToken` should implement `Display`, especially since `RefreshToken` already does.